src/main/daml/Daml/Finance/Claims/Util/Define.daml

```diff
@@ -1,7 +1,7 @@
 -- Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
 -- SPDX-License-Identifier: Apache-2.0
 
-module Daml.Finance.Instrument.Generic.Util where
+module Daml.Finance.Claims.Util.Define where
```

Comment:
What do you think about 

```suggestion
module Daml.Finance.Claims.Util.Builders where
```?

No strong opinion from my side

Reply:
Agree, we use `Builders` for these functions in CC as well.

Reply:
I have renamed it now.